frontend/src/components/ExternalResourceModal.js

```diff
@@ -0,0 +1,104 @@
+import React, { useCallback, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button, Modal, Alert, useModal, connectModal,
+} from '@trussworks/react-uswds';
+
+import { isValidURL } from '../utils';
+
+const ESCAPE_KEY_CODE = 27;
+
+const ExternalResourceModal = ({ onOpen, onClose }) => (
+  <Modal
+    title={<h3>External Resources Disclaimer</h3>}
+    actions={(
+      <>
+        <Button type="button" onClick={onOpen}>
+          View External Resource
+        </Button>
+
+        <Button type="button" secondary onClick={onClose}>
+          Cancel
+        </Button>
```

Comment:
Can you switch the order of these buttons and change the colors to match the file delete modal?

Reply:
I switched the order but from the code it looks like the colors should match without doing anything? I can't test locally since I get an error when trying to upload a file (most likely cause I don't have the other services running locally) and I don't have permissions in staging to use the app to view it there

Reply:
Yep colors look to be correct to me